recipes/packages.rb

```diff
@@ -15,7 +15,17 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-include_recipe 'yum-ius' if node['osl-php']['use_ius']
+if node['osl-php']['use_ius']
+  include_recipe 'yum-ius'
+
+  if node['php']['version'].to_f == 7.1
+    node['yum'].each_key do |repo|
```

Comment:
This should be ``node['yum-ius']['repos']``. See [here](https://github.com/chef-cookbooks/yum-ius/blob/master/recipes/default.rb#L22-L30) as an example.

Reply:
I get a `undefined method `[]' for nil:NilClass` error when I use that attribute.

Reply:
Ah, it looks like the newer cookbook does that but the one we have locked uses a different method. I would change it to ``node['yum-ius']['ius']`` based on [this](https://github.com/chef-cookbooks/yum-ius/blob/v0.4.6/recipes/default.rb#L19-L66) which is the version we have. You'll need to fix the logic here too since this isn't a list of repos, but the attribute for the ius repo (which is the only repo I think we install. The other method is to use the manual loop that upstream has if we want to be safe.

Reply:
Made the changes discussed in the sprint meeting!